site/docs/reference/Connectors/capture-connectors/hubspot-native.md

```diff
@@ -0,0 +1,116 @@
+# Hubspot ( Native )
+
+This connector captures data from Hubspot into Flow collections. 
+
+
+## Warning
+This connector has been developed by Estuary's team, and will differ from the imported Hubspot connector.
```

Comment:
I agree I dont think this is a useful warning to have at the top of the documentation. A warning implies that caution should be used before proceeding which isn't necessarily true. If anything I would encourage people to use this connector instead of the Airbyte import

Reply:
Removed warning [here](1c570e7d64adfc904efb31b925cb50f4d45b0d43)